cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/model/support/ldap/LdapAuthenticationProperties.java

```diff
@@ -40,6 +40,7 @@
     private boolean usePasswordPolicy = true;
     private String dnFormat;
     private String principalAttributeId = "uid";
+    private String principalAttributePassword = "userPassword";
```

Comment:
You'll need to also document this property, and briefly explain what its purpose it perhaps in the docs where properties are all listed.